sqs_with_iam/output.tf

```diff
@@ -1,24 +1,25 @@
 output "arn" {
-  value       = ["${aws_sqs_queue.queue.*.arn}"]
+  value       = [aws_sqs_queue.queue.*.arn]
```

Comment:
Are these brackets needed as the splat will make it a list?

https://www.terraform.io/upgrade-guides/0-12.html#referring-to-list-variables